src/main/java/de/unijena/cheminf/mortar/model/util/CollectionUtil.java

```diff
@@ -48,78 +48,73 @@ private CollectionUtil() {
     //</editor-fold>
     //
     //<editor-fold desc="public static methods" defaultstate="collapsed">
-    //TODO: check parameters, change params to enum values (and bools?), throw exceptions!
     /**
-     * Sorts given list by property and sort type.
+     * Sorts a given list of Molecule-/FragmentDataModel objects according to the specified property and
+     * the sort type specified by a Boolean. If true, the list is sorted in ascending order,
+     * otherwise in descending order.
      *
-     * @param aList List
-     * @param aProperty String
-     * @param aSortType String
+     * @param aList List of Molecule-/FragmentDataModel objects to be sorted.
+     * @param aProperty String property of Molecule-/FragmentDataModel by which to sort the list.
```

Comment:
Define that the given string must correspond to the text property of a DataModelPropertiesForTableView enum constant.

Reply:
It does not have to, only if, the list is sorted

Reply:
You are throwing an IllegalArgumentException if it does not, or am I getting this wrong?

Reply:
You are right, but this is described in the @throw tag